python_docs_theme/layout.html

```diff
@@ -47,6 +47,12 @@
     <div class="footer">
     &copy; <a href="{{ pathto('copyright') }}">{% trans %}Copyright{% endtrans %}</a> {{ copyright|e }}.
     <br />
+    {% trans %}This page is licensed under the Python Software Fundation License Version 2.{% endtrans %}
+    <br />
+    {% trans %}Any code contained in this page is additionally licensed under the Zero Clause BSD License.{% endtrans %}
```

Comment:
If you look at https://docs.python.org/dev/library/functools.html there is a link "Source code: Lib/functools.py" which points to Lib/functools.py file. This file is not licensed as BSD0 and will not be licensed as BSD0.

"Any code contained in this page" is a little bit unclear. "examples, recipes, and other code in the documentation" sounds more precise: https://github.com/python/cpython/pull/17635/files

Reply:
Done